flex-checks/src/main/java/org/sonar/flex/checks/ConditionalStructure.java

```diff
@@ -59,6 +59,9 @@ static BranchAndContent branchAndContentSwitch(AstNode branch, List<AstNode> dir
   }
 
   boolean areAllEquivalentBranches() {
+    if (branches.isEmpty()) {
+      return true;
```

Comment:
@amelie-renard-sonarsource I would put `false` when there are no branches as there are no equal branches.
```suggestion
      return false;
```

Reply:
Yes, I hesitated...